src/ascci_utils.py

```diff
@@ -0,0 +1,66 @@
+from typing import Tuple
+
+from asciimatics.effects import Print
+from asciimatics.exceptions import StopApplication
+from asciimatics.scene import Scene
+from asciimatics.screen import Screen
+
+from asciimatics_mod import ColourImageFilePIL
+from CamReader import CamReader
+
+
+class ascci:
+    """Generic ascii methods"""
+
+    def __init__(self) -> None:
+        pass
```

Comment:
won;t it be better if we assign screen  as a property of self ?

Reply:
yes, if we decide to use ascci_utils for screen creation.